plugins/modules/win_route.py

```diff
@@ -11,16 +12,21 @@
 description:
     - Add or remove a static route.
 options:
-  destination:
+  network:
```

Comment:
You cannot change a module option name as that will break existing users of the module. If `network` is a desired name you will either have to make it an alias for `destination` or vice versa. This allows people to continue to use the older option as well as the newer one at the same time.